lib/zbuff.h

```diff
@@ -45,29 +45,40 @@ extern "C" {
 *  Compiler specifics
 *****************************************************************/
 /*!
+*  ZSTD_DLL_STDCALL :
+*  Provide the ability to use stdcall linkage on public interface when building a Windows DLL
```

Comment:
I'm not an expert of this domain,
but shouldn't `stdcall` be the default calling convention when compiling for Windows anyway ?